impressionist.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency "friendly_id"
   s.add_dependency 'nokogiri', RUBY_VERSION < '2.1.0' ? '~> 1.6.0' : '~> 1'
-  s.add_dependency 'rails', '>= 3.2.15', '< 7.1'
+  s.add_dependency 'rails', '>= 6.1.7.6', '< 7.1.2'
```

Comment:
If these changes allow us to support Rails 7.1, I wonder if we can just remove the second constraint completely?

```ruby
  s.add_dependency 'rails', '>= 6.1.7.6'
```

Reply:
yep, great call, thank you!

changed it in [a69b466](https://github.com/ConvertKit/impressionist/pull/3/commits/a69b466e7df135abeddb9e387572ff4d2a607891)